src/main/resources/application.yml

```diff
@@ -1,7 +1,19 @@
 server:
-  port: 8000
+  port: 9000
 hdaf:
   clientregistry:
-    XAuthToken: ${Auth_Token}
+    XAuthToken: 8e983a1719e20913be6c61e9b552e88fc6032c21e1030f1abda986d17326b2b0
     clientId: 6
-    url: ${Client_Registry_Url}
+    url: http://localhost:8080/api/1.0
+  userservice:
+    XAuthToken: 8e983a1719e20913be6c61e9b552e88fc6032c21e1030f1abda986d17326b2b0
```

Comment:
There will be ClientID and Client Password, which will be used in the login API to get auth token? right now it's not required, we can just remove XAuthToken also. and then follow proper naming convention?